js/card.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+window.card = (function () {
+  var cardTemplate = document.querySelector('#card').content;
+  var map = document.querySelector('.map');
+  var mapFiltersContainer = map.querySelector('.map__filters-container');
+
+  var typesOfHouse = {
+    flat: 'Квартира',
+    bungalo: 'Бунгало',
+    house: 'Дом',
+    palace: 'Дворец',
+  };
+
+  function renderCard(add) {
+    var photosFragment = document.createDocumentFragment();
+    var featuresFragment = document.createDocumentFragment();
+    var cloneCard = cardTemplate.cloneNode(true);
+    var titleElement = cloneCard.querySelector('.popup__title');
+    var addressElement = cloneCard.querySelector('.popup__text--address');
+    var priceElement = cloneCard.querySelector('.popup__text--price');
+    var typeElement = cloneCard.querySelector('.popup__type');
+    var capacityElement = cloneCard.querySelector('.popup__text--capacity');
+    var timestampsElement = cloneCard.querySelector('.popup__text--time');
+    var descriptionElement = cloneCard.querySelector('.popup__description');
+    var avatarElement = cloneCard.querySelector('.popup__avatar');
+    var photosElement = cloneCard.querySelector('.popup__photos');
+    var featuresElement = cloneCard.querySelector('.popup__features ');
+
+    if (add.offer.title) {
+      titleElement.textContent = add.offer.title;
+    } else {
+      titleElement.remove();
+    }
+
+    if (add.offer.address) {
+      addressElement.textContent = add.offer.address;
+    } else {
+      addressElement.remove();
+    }
+
+    if (add.offer.price) {
+      priceElement.textContent = add.offer.price + '₽/ночь';
+    } else {
+      priceElement.remove();
+    }
+
+    if (add.offer.type) {
+      typeElement.textContent = typesOfHouse[add.offer.type];
+    } else {
+      typeElement.remove();
+    }
+
+    if (add.offer.capacity) {
```

Comment:
тут точно нужен capacity?
```suggestion
    if (add.offer.rooms && add.offer.guests) {
```